test/parallel/test-bootstrap-modules.js

```diff
@@ -1,14 +1,27 @@
-// Flags: --expose-internals
 'use strict';
 
-// This list must be computed before we require any modules to
+// This list must be computed before we require any builtins to
 // to eliminate the noise.
-const actualModules = new Set(process.moduleLoadList.slice());
+const list = process.moduleLoadList.slice();
 
 const common = require('../common');
 const assert = require('assert');
+const { inspect } = require('util');
 
-const expectedModules = new Set([
+const preExecIndex =
+  list.findIndex((i) => i.includes('pre_execution'));
+const actual = {
+  beforePreExec: new Set(list.slice(0, preExecIndex)),
+  atRunTime: new Set(list.slice(preExecIndex, list.length)),
+};
+
+// Currently, we don't add additional builtins to worker snapshots.
+// So for worker snapshots we'll just concatenate the two. Once we
+// add more builtins to worker snapshots, we should also distinguish
+// the two stages for them.
+const expected = {};
+
+expected.beforePreExec = new Set([
```

Comment:
To clearly indicate that these modules were executed at build time, I would find a name like `atStartupSnapshot` or `atBuildTime`, to be more intuitive as contradicting `atRunTime`.

Reply:
I thought about that but it seems weird until workers do load builtins into the snapshot (which was what I was working on originally). For now it seems less weird to call it `beforePreExec` which is still factually correct. But I am also okay with using `atStartupSnapshot` and add a note saying that this is not factually correct for workers yet.